packages/atlas/src/components/_inputs/Slider/RatioSlider.tsx

```diff
@@ -14,7 +15,7 @@ type RatioSliderProps = {
 }
 
 export const RatioSlider = forwardRef<HTMLInputElement, RatioSliderProps>(
-  ({ min = 0, max = 100, step = 10, defaultValue = 50, value: controlledValue, onChange }, ref) => {
+  ({ min = 0, max = 100, step = 10, defaultValue = 50, value: controlledValue, onChange, id = '' }, ref) => {
```

Comment:
Setting an id is a good fix but IMO most dev would expect it to be set on the `<input id={id}` too.

Also for when the `id` isn't set I think it should not get a default value. Instead:
```js
const internalId = useMemo(() => id ?? createId(), [id])
```
could be used on:
```jsx
<mask id={`cutout-mask-${internalId}`}>
```